src/libcollectionstest/cow_str.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2012-2013-2014 The Rust Project Developers. See the COPYRIGHT
```

Comment:
You'll need to add `mod cow_str;` to `src/libcollectionstest/lib.rs` for this file to be used.

Also I'm pretty sure "2012-2013-2014" isn't the correct year 😉.